client/state/partner-portal/partner/reducer.ts

```diff
@@ -62,6 +62,15 @@ const current = ( state = initialState.current, action: AnyAction ) => {
 			}
 
 			return action.partner;
+
+		case 'STORED_CARDS_UPDATE_IS_PRIMARY_COMPLETED':
```

Comment:
I've missed to notice we're using a string for the action type - let's use a constant as we do for all other actions.